### PR TITLE
test(artifact test): added amazon linux 2 arm artifact test

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: configurations/arm/amazon2.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ami_id_db_scylla: 'ami-06b4c3e5d2605128e'  # Amazon Linux 2 LTS Arm64 AMI 2.0.20220606.1 arm64 HVM gp2
+region_name: 'eu-west-1'
+instance_type_db: 'im4gn.xlarge'
```
Comment (fgelcer) [inline configurations/arm/amazon2.yaml:3]: in the original file, the instance type is `i3.large`.. is this the equivalent for ARM?

Comment (ShlomiBalalis) [reply configurations/arm/amazon2.yaml:3]: Yes. For reference, you can take a look at [artifacts-ami-test-arm](https://jenkins.scylladb.com/view/master/job/scylla-master/job/artifacts/job/artifacts-ami-test-arm/76/parameters/). The x86 one uses `i3.large`, and the arm one uses `im4gn.xlarge` instead.

Comment (fgelcer) [reply configurations/arm/amazon2.yaml:3]: yes, i saw that... that's why i'm asking... wouldn't the parallel of that `i3.large` be `im4gn.large` ?

Comment (fgelcer) [reply configurations/arm/amazon2.yaml:3]: other than this question, and failed mocked_tests it LGTM

Comment (ShlomiBalalis) [reply configurations/arm/amazon2.yaml:3]: It's what roy recommended back when we first started to support arm
https://github.com/scylladb/scylla-pkg/issues/2744#issuecomment-1034782347
@roydahan Thoughts?

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by enaydanov — APPROVED** [review]

**Pull request merged**